docs/advanced-config.md

```diff
@@ -16,27 +16,21 @@
 | `OTEL_TRACES_EXPORTER`            | `exporter_factories`  | `otlp`                           | Stable      | Select the traces exporter to use. We recommend using either the OTLP exporter (`otlp`) or the Jaeger exporter (`jaeger-thrift-splunk`).
 
 The Splunk Distribution of OpenTelemetry Python uses the OTLP traces exporter as the default setting. Please note that the
-OTLP format is not supported by the (now deprecated) [SignalFx Smart Agent](https://github.com/signalfx/signalfx-agent).
-If you wish to use the Jaeger exporter instead, you can set it by using the `OTEL_TRACES_EXPORTER` environment variable.
+OTLP format is neither supported by Splunk ingest nor by the (now deprecated) [SignalFx Smart Agent](https://github.com/signalfx/signalfx-agent).
```

Comment:
nit: what do you think of replacing the term `Splunk ingest` to `[Splunk Observability Cloud](https://dev.splunk.com/observability/docs/apibasics/api_list/)` ?

Taken from https://github.com/signalfx/splunk-otel-go#splunk-specific-configuration